parsing-scripts/portindex2fixtures.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+with open('portindex.json', "r") as file:
+    data = json.load(file)
+
+i = 0
+port_fixtures = []
+for port in data:
+    maintainers = []
+    one_maintainer = []
+    for maintainer in port['maintainers']:
+        try:
+            one_maintainer.append(maintainer['email']['name'])
+            one_maintainer.append(maintainer['email']['domain'])
+        except:
+            continue
+        maintainers.append(one_maintainer)
+        one_maintainer = []
+    port['maintainers'] = maintainers
+
+
+    newdata = {
+        'model': 'ports.Port',
+        'pk': i,
+        'fields': port
+    }
+    port_fixtures.append(newdata)
+    i = i + 1
+
+with open('../port_fixtures.json', 'w') as newjson:
```

Comment:
Just curious: Why do you need to do this conversion? Couldn't this go to the database directly?

Anyway, please remove port_fixtures.json, that one should not be part of repository, it's simply too big and always outdated.

Reply:
> Just curious: Why do you need to do this conversion? Couldn't this go to the database directly?

It could go. But django fixtures are more efficient and they handle relations very well [for inserting initial data]. Updating can then be done directly as the data won't be very large.

But I guess, we could avoid some complexity by skipping fixtures and going for entering data directly. Thank You